---
title: LOCALDB_ERROR_INTERNAL_ERROR | Microsoft Docs
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: 60c7a47b-114c-441b-869e-9d8ebfe4148e
author: stevestein
ms.author: sstein
ms.openlocfilehash: fc75b2c016389b850c785462b75aa0b5acdaee07
ms.sourcegitcommit: 216f377451e53874718ae1645a2611cdb198808a
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 07/28/2020
ms.locfileid: "87246030"
---
# <a name="localdb_error_internal_error"></a>LOCALDB_ERROR_INTERNAL_ERROR
 [!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]
    
## <a name="details"></a>Detalhes  
  
| Atributo | Valor |
| --------- | ----- |
|Nome do Produto|SQL Server|  
|ID do evento|264|  
|Origem do Evento|Runtime de banco de dados local do SQL Server 12.0|  
|Componente|API do runtime de banco de dados local|  
|Texto da mensagem|Ocorreu um erro inesperado dentro de uma chamada de método API de instância de banco de dados local. Consulte o log do Aplicativo do Windows para obter detalhes sobre o erro.|  
  
## <a name="explanation"></a>Explicação  
 Erro inesperado.  
  
## <a name="user-action"></a>Ação do usuário  
 Consulte o log de eventos para obter detalhes.  
  
  
